aten/src/ATen/native/native_functions.yaml

```diff
@@ -961,10 +973,14 @@
 - func: logical_or(Tensor self, Tensor other) -> Tensor
   device_check: NoCheck   # TensorIterator
   variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: logical_or
 
 - func: logical_or_(Tensor(a!) self, Tensor other) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
   variants: method
+  dispatch:
+    CompositeExplicitAutograd: logical_or_
```

Comment:
This PR fixes the root cause of the composite compliance test failure for these operators (https://github.com/pytorch/pytorch/blob/aa2163eba53036d998d564138e10f18634c890b9/torch/testing/_internal/common_methods_invocations.py#L9707-L9708). You'll probably have to remove those expected failures